.github/workflows/lint.yml

```
@@ -38,7 +38,7 @@ jobs:
       - name: Run Biome-Lint
         run: npm run biome-ci
         id: biome_lint
-        continue-on-error: true
+        # continue-on-error: true
```

```suggestion
        continue-on-error: true
```
With the annotations working I think it's fine to continue on error

---

yeah, but I wanted to make sure that we aren't seeing any eslint annotations by accident